robotiq_2f_gripper_action_server/package.xml

```diff
@@ -23,4 +23,6 @@
   <depend>robotiq_2f_gripper_control</depend>
   <depend>roscpp</depend>
 
+  <build_depend>message_generation</build_depend>
```

Comment:
Should `<exec_depend>message_runtime</exec_depend>` also be added?

Reply:
Actually even the message_generation dependency was not needed, this package is not generating messages. I will remove it here and for the CMakeLists

Thanks!